tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java

```diff
@@ -63,6 +68,11 @@ public long getCount() {
 
     @Override
     public void run() {
+      try {
+        mBarrier.await();
+      } catch (InterruptedException | BrokenBarrierException e) {
+        e.printStackTrace();
```

Comment:
a slightly cleaner way to do this is to make all tasks implementing `Callable` rather than `Runnable`, which can throw exceptions.

See http://stackoverflow.com/questions/141284/the-difference-between-the-runnable-and-callable-interfaces-in-java and an example code in alluxio: https://github.com/Alluxio/alluxio/pull/3611/files